pages/api/verify/[uniqueString].js

```diff
@@ -0,0 +1,22 @@
+import nc from 'next-connect';
+import userService from '../../../src/server/services/user.service';
+
+const handler = nc({
+})
+  .get(async (req, res) => {
+    try {
+      const { uniqueString } = req.query;
+      const user = await userService.findUniqueString(uniqueString);
```

Comment:
There would be enough to have a single service method to find and confirm email. No need in calling `findUniqueString` and `addConfirmation` separately.